love-letter-app/src/app/globals.css

```diff
@@ -31,3 +31,38 @@ body {
     text-wrap: balance;
   }
 }
+
+.create-card-page{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.create-card-container{
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(59, 59, 59);
+  border-radius: 10px;
+  padding: 15px;
+  width:90%;
+  max-width: 600px;
+}
+
+.svg-container{
+  background-color: black;
```

Comment:
consider using a variable if this background color and message-containers background color are the same, unless this is just temporary